torch/_dynamo/pgo.py

```diff
@@ -1,25 +1,37 @@
 from __future__ import annotations
 
+import base64
 import copy
 import dataclasses
 import enum
 import logging
+import os
+import pickle
 import time
 from collections import defaultdict
 from typing import DefaultDict, Optional, Tuple, TYPE_CHECKING, TypeVar, Union
 from typing_extensions import Self
 
+import torch._dynamo.config
+import torch._utils_internal
+import torch.compiler.config
 from torch._dynamo.utils import get_chromium_event_logger
+from torch._environment import is_fbcode
+from torch._inductor.remote_cache import create_cache
```

Comment:
Doesn't have to be a part of this PR but does it make sense to extract out remote_cache into `torch._remote_cache` or something that's not coupled with inductor? Seems like it's really just a generic remote KV store interface at this point right?

Reply:
yes. There are several file moves that I haven't done but I may do as prep PRs and stack under this one.